src/sentry/runner/commands/devservices.py

```diff
@@ -24,9 +25,8 @@
 # assigned as a constant so mypy's "unreachable" detection doesn't fail on linux
 # https://github.com/python/mypy/issues/12286
 DARWIN = sys.platform == "darwin"
-COLIMA = os.path.expanduser("~/.local/share/sentry-devenv/bin/colima")
```

Comment:
[same comment](https://github.com/getsentry/sentry/pull/67533/files#r1547008829) also applies to here